spec/models/user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { described_class.new(name: 'John') }
+
+  describe 'associations' do
+    it { should have_many(:posts).with_foreign_key(:author_id).dependent(:destroy) }
+    it { should have_many(:comments).with_foreign_key(:author_id) }
```

Comment:
- **[OPTIONAL]** This test returns an error, please fix it to pass them all.
![image](https://github.com/Zeph180/blog-it/assets/58642949/3733b347-5993-4c5b-b04e-dc3f3ffa3959)